src/web_app_skeleton/src/actions/errors/show.cr.ecr

```diff
@@ -4,126 +4,49 @@ class Errors::Show < Lucky::ErrorAction
   <%- else -%>
   default_format :html
   <%- end -%>
+  dont_report [Lucky::RouteNotFoundError]
 
-  def render(error : Lucky::UnknownAcceptHeaderError)
-    message = "Unrecognized Accept header '#{request.headers["Accept"]?}'."
-
-    <%- if api_only? -%>
-    json ErrorSerializer.new(message), status: 406
-    <%- else -%>
-    if json?
-      json ErrorSerializer.new(message), status: 406
-    else
-      render_error_page status: 406, title: message
-    end
-    <%- end -%>
-  end
-
-  def render(error : Lucky::NotAcceptableError)
-    message = "Accept header '#{request.headers["Accept"]?}' is not accepted."
-
-    <%- if api_only? -%>
-    json ErrorSerializer.new(message), status: 406
-    <%- else -%>
-    if json?
-      json ErrorSerializer.new(message), status: 406
-    else
-      render_error_page status: 406, title: message
-    end
-    <%- end -%>
-  end
-
-  def render(error : Lucky::ParamParsingError)
-    message = "There was a problem parsing the JSON params." +
-              " Please check that it is formed correctly."
-
-    <%- if api_only? -%>
-    json ErrorSerializer.new(message), status: 400
-    <%- else -%>
-    if json?
-      json ErrorSerializer.new(message), status: 400
-    else
-      render_error_page status: 500
-    end
-    <%- end -%>
-  end
-
-  def render(error : Lucky::Exceptions::MissingNestedParam)
-    <%- if api_only? -%>
-    json ErrorSerializer.new("Missing param key: '#{error.nested_key}'"), status: 400
-    <%- else -%>
+  def render(error : Lucky::RenderableError)
     if json?
-      json ErrorSerializer.new("Missing param key #{error.nested_key}"), status: 400
+      error_json error.renderable_message, status: error.renderable_status
     else
-      render_unhandled_error(error)
+      error_html status: error.renderable_status
```

Comment:
For most errors we want the default "Something went wrong"  for HTML and not something super dev specific like "Missing nested params"